src/views/Invite/Cohort.js

```diff
@@ -1,398 +1,397 @@
-import React, { useMemo, useEffect, useCallback, useState } from 'react';
-import { Grid, Flex, Button } from 'indigo-react';
-import * as ob from 'urbit-ob';
-import { Just, Nothing } from 'folktale/maybe';
+import React, { useEffect, useCallback, useState } from 'react';
```

Comment:
This file you inherited is rather large, is it possible to refactor some of the conditional branches (e.g., the `hasPending` or `networkKeysNotSet` paths) and / or the component itself into smaller submodules? Would make this much more manageable for future reviewers and maintainers. Will defer to you on whether or not to address this now, it might be more trouble than its worth.

Reply:
I agree. I might be able to do it tonight.